functions/Get-PublicIP.ps1

```diff
@@ -0,0 +1,43 @@
+function Get-PublicIP {
+	<#
+	.SYNOPSIS
+		Returns the public IP address for the connected network.
+	.DESCRIPTION
+		Uses the ipify.org API to lookup the Public IP address.
+	.EXAMPLE
+		C:\PS> Get-PublicIP
+		Returns the current IPv4 Public IP for the connected network.
+	.EXAMPLE
+		C:\PS> Get-PublicIP -v6
+		Returns the current IPv6 (if available) Public IP for the connected network.
+	.OUTPUTS
+		Public IP address(es)
+	.NOTES
+		Version 1.0.0
+	#>
+	[CmdletBinding()]
+	param (
+		# IPv6 switch
+		[Parameter(Mandatory = $false, HelpMessage = 'Perform an IPv6 lookup')]
+		[switch] $v6
+	)
+
+	begin {
+		$ErrorActionPreference = 'Stop'
+		$Uri = 'https://api.ipify.org?format=json'
+		if ($6) {
+			$Uri = 'https://api64.ipify.org?format=json'
+		}
+	}
+
+	process {
+		$resp = (Invoke-WebRequest $Uri -ErrorAction 'stop').Content | ConvertFrom-Json
```

Comment:
Add a line under the $resp

$resp.ip | clip

Reply:
Done this in the end block now.